### PR TITLE
An attempt to summerise ledger errors

### PR DESCRIPTION
**Pull request by remon-nashid**
Just a lame attempt to categorize ledger error status into one of three:
- `Device`: ie device is locked, not plugged, not permissioned.
- `App`: ie app not open or not installed.
- `Ok`: means we're ready to import address from ledger or sign via ledger.

### CODE REVIEW

**Review by wahidrahim — COMMENTED** [review]
Looks like github actions are failing due to: `packages/ui/src/Popup/Accounts/index.tsx#L137`

Could it just be written as:

```
isLedgerCapable ? 'Attach ledger account' : 'Ledger devices can only be connected with Chrome browser'
```
?

File: packages/ui/src/hooks/useLedger.ts [hunk 1]
```diff
@@ -74,6 +74,10 @@ export function useLedger (genesis?: string | null, accountIndex = 0, addressOff
     } else if (error?.includes('AbortError: The transfer was cancelled') ||
     error?.includes('No device selected') ||
     error?.includes("Failed to execute 'requestDevice' on 'USB': Must be handling a user gesture to show a permission request") ||
+    // Strangely enough this error is thrown even while importing an account.
+    // @FIXME distinguish it from actual tx rejection, once we're able to sign with ledger.
+    error?.includes('Error: Transaction rejected') ||
```
Comment (wahidrahim) [inline packages/ui/src/hooks/useLedger.ts:79]: Yup.

From when I noticed, this error showed up after I had the Polymesh app open, and then the Ledger locked by itself from idling. After unlocking the Ledger, the Polymesh app was still open, and everything was working as expected.

Need to confirm by reproducing consistently though.

**Review by wahidrahim — APPROVED** [review]
Looks great!

Seems like the e2e issue is due jest failing to load ES modules from Polkadot packages; might have to modify the jest config for e2e perhaps

**Comment by remon-nashid** [comment]
e2e is fixed.

**Pull request merged**